python/tk_multi_workfiles/file_search_cache.py

```diff
@@ -115,7 +114,7 @@ def find_file_versions(self, work_area, file_key, clean_only=False):
             return {}
 
         # return a dictionary of version:FileItem entries:
-        return dict([(v, f) for v, f in six.iteritems(file_info.versions)])
+        return dict([(v, f) for v, f in file_info.versions.items()])
```

Comment:
if `file_info.versions` is already a dictionary, why do we need to map it to a new dictionary?

Will it work?

```suggestion
        return file_info.versions
```

Maybe I'm missing something...

Reply:
Maybe we want to send a copy of the object instead of a reference?!